lib/_utils_multiple-choice.ts

```diff
@@ -0,0 +1,41 @@
+import { OptionType } from "../components/predict-form/PredictProvider"
+
+export function normalizeOptionsToHundred(options: OptionType[]) {
```

Comment:
considered naming getNormalizeOptionsToHundred to make it clearer we're not mutating – let me know what you think. I'm usually a big "get prefixer" but I can see the good in both approaches